api/v1_projects.inc

```diff
@@ -104,10 +104,16 @@ function api_v1_projects($method, $data, $query_params)
 
     api_send_pagination_header($query_params, $total_rows, $per_page, $page);
 
+    // restrict to list of desired fields, if set
+    $return_fields = array_get($query_params, "fields", null);
```

Comment:
Not really relevant to this MR but I came across the null coalescing operator recently which does exactly the same as array_get so we could write `$return_fields = $query_params["fields"] ?? null;` in native php.